Tests/backend-specific-tests/Teradata/RunnerTeradataTest.php

```diff
@@ -96,16 +77,13 @@ public function testWorkspaceSynapseMapping()
                 'definition' => [
                     'type' => 'aws-ecr',
                     'uri' => '061240556736.dkr.ecr.us-east-1.amazonaws.com/keboola.runner-workspace-test',
-                    'tag' => 'latest',
+                    'tag' => 'dev-erik-PST-626.1',
```

Comment:
```suggestion
                    'tag' => 'latest',
```
Čeká na https://github.com/keboola/runner-workspace-test/pull/24

Reply:
fixed